cfgov/unprocessed/apps/youth-employment-success/css/main.less

```diff
@@ -288,6 +290,10 @@ input:-moz-ui-invalid {
 }
 
 .js-transit-hours, .js-transit-minutes {
+  // Necessary because we are setting a lot of custom widths per design's
```

Comment:
Feel empowered to push back on design if needed since adhering to conventions elsewhere on the site > customizations.

Reply:
Yeah, we changed designers mid-stream and I did try to push back, but they are insistent on this point, I guess.